README.md

```diff
@@ -1,3 +1,31 @@
 # FusionAuth Example React App With SDK
-This is an example react app made with [https://github.com/FusionAuth/fusionauth-react-sdk](FusionAuth's React SDK)
-Currently under development.
\ No newline at end of file
+This is an example react app made with [FusionAuth's React SDK](https://github.com/FusionAuth/fusionauth-react-sdk).
+
+## Prerequisites
+You will need to have the following things installed on your computer:
+ - [Git](https://git-scm.com/): This is will used to `git clone` the repo.
```

Comment:
No need to list Git as a pre-requisite, since that is pretty much assumed if you are cloning something from GitHub